### PR TITLE
[RM-23534] Fix distro detection for Arch Linux

### PR DESCRIPTION
**Pull request by HacKanCuBa**
I'm not sure what changed in Arch in the last 2 months since my last commit, but it's an easy fix :)

### CODE REVIEW

File: ceph_deploy/hosts/remotes.py [hunk 1]
```diff
@@ -36,10 +36,12 @@ def platform_information(_linux_distribution=None):
                 codename = minor
             else:
                 codename = major
-    if not codename and 'oracle' in distro.lower(): # this could be an empty string in Oracle linux
+    if not codename and 'oracle' in distro.lower():  # this could be an empty string in Oracle linux
```
Comment (vasukulkarni) [inline ceph_deploy/hosts/remotes.py]: accidentally change?, same for the next line below?

**Comment by vasukulkarni** [comment]
Jenkins test this please

File: ceph_deploy/hosts/remotes.py (hunk 1 above)
Comment (HacKanCuBa) [reply ceph_deploy/hosts/remotes.py]: yes, auto pep8 added an additional space to the comment

**Comment by djgalloway** [comment]
jenkins test this please

**Comment by djgalloway** [comment]
jenkins test this please

**Comment by djgalloway** [comment]
jenkins test this please

**Comment by alfredodeza** [comment]
jenkins test this please

**Pull request merged**